contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -21,13 +21,24 @@ pragma solidity 0.4.18;
 import "zeppelin-solidity/contracts/math/SafeMath.sol";
 import "../DebtRegistry.sol";
 import "../TermsContract.sol";
+import "ethereum-datetime/contracts/DateTime.sol";
 
 
 contract SimpleInterestTermsContract is TermsContract {
     using SafeMath for uint;
 
     enum AmortizationUnitType { HOURS, DAYS, WEEKS, MONTHS, YEARS }
 
+    struct SimpleInterestParams {
+        uint principalPlusInterest;
+        uint startTimestamp;
+        uint endTimestamp;
+        AmortizationUnitType amortizationUnitType;
+        uint amortizationUnitLengthInSeconds;
+        uint termLengthInAmortizationUnits;
+        uint termLengthInSeconds;
```

Comment:
Maybe I'm missing something, but why do `termLengthInSeconds` and `amortizationUnitLengthInSeconds` need to get passed around in this internal struct?

They only seem to be used as intermediate values in calculations for figuring out the `endTimestamp`

Reply:
Yes, you're correct. I was putting anything and everything into the struct while trying to determine where things were going wrong in my tests. Will remove.